delayed-dashboard/src/main/java/dashboard/DelayedDashboardTask.java

```diff
@@ -0,0 +1,88 @@
+package dashboard;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.WebSocket;
+import io.vertx.core.json.JsonObject;
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.concurrent.Task;
+
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.TimeUnit;
+
+public class DelayedDashboardTask extends Task<Void> {
+
+   static final String HTTP_HOST = System.getProperty("http.host", "station-boards-injector-myproject.127.0.0.1.nip.io");
+   static final int HTTP_PORT = Integer.getInteger("http.port", 80);
+
+   private ObservableList<DelayedTrainView> partialResults =
+         FXCollections.observableArrayList();
+
+   private Vertx vertx = Vertx.vertx();
+   private HttpClient client = vertx.createHttpClient();
+
+   private BlockingQueue<DelayedTrainView> queue = new ArrayBlockingQueue<>(128);
```

Comment:
I can't see where the queue gets data

Reply:
Great catch, this doesn't seem to be no longer in use. Instead the `ObservableList` above is used which is a Java FX collection.